### PR TITLE
User 2_1st set of Changes

### PR DESCRIPTION
**Pull request by praveenkpsastraauto**
Hey, 

Pls approve this

Regards,
User 2

### CODE REVIEW

File: src/test/java/Package_User2/user2Changes.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package Package_User2;
+
+public class user2Changes {
+	
+	public static void main(String[] args)
+	{
+		System.out.println("User 2 Changes added");
+
```
Comment (praveen1993kp) [inline src/test/java/Package_User2/user2Changes.java:9]: Add User Name

File: target/test-classes/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/Package1/
+/Package2/
+/Package_User2/
```
Comment (praveen1993kp) [inline target/test-classes/.gitignore:3]: Okay

**Review by praveen1993kp — CHANGES_REQUESTED** [review]
Hey,

Make these changes asap

File: src/test/java/Package_User2/user2Changes.java (hunk 1 above)
Comment (praveenkpsastraauto) [reply src/test/java/Package_User2/user2Changes.java:9]: User Name added

File: target/test-classes/.gitignore (hunk 2 above)
Comment (praveenkpsastraauto) [reply target/test-classes/.gitignore:3]: Okay

**Review by praveenkpsastraauto — COMMENTED** [review]
okay

**Comment by praveenkpsastraauto** [comment]
Changes done. Pls check

**Review by praveen1993kp — COMMENTED** [review]
sss

**Comment by praveenkpsastraauto** [comment]
dsd

**Review by praveen1993kp — APPROVED** [review]
Approved

**Pull request merged**